doc/cookbook/source/examples/base_api/dense_dispatching.rst

```diff
@@ -0,0 +1,25 @@
+==================================================
+Dense Dispatching for Linear Discriminant Analysis
+==================================================
+
+Some algorithms like Linear Discriminant Analysis, Least Angle Regression, etc. in shogun support training with features of multiple primitive type.
+
+-------
+Example
+-------
+
+Imagine we have files with training data. We create 64 bit and 32 bit float CDenseFeatures of appropriate primitive type and also create :sgclass:`CBinaryLabels` as
+
+.. sgexample:: dense_dispatching.sg:create_features
+
+We create an instance of :sgclass:`CLDA` and provide labels.
+
+.. sgexample:: dense_dispatching.sg:create_instance
+
+We train with 64 bit RealFeatures as
```

Comment:
Group all of the feature calls together. And say "We can pass any of the feature type to the algorithm for training"